src/bundle/Resources/public/js/CKEditor/custom-attributes/ui/custom-attributes-form-view.js

```diff
@@ -146,20 +146,22 @@ class IbexaCustomAttributesFormView extends View {
             children.add(classesView);
         }
 
-        Object.entries(customAttributes).forEach(([name, config]) => {
-            const createAttributeMethod = this.attributeRenderMethods[config.type];
+        if (customAttributes && Object.keys(customAttributes).length !== 0) {
```

Comment:
```suggestion
        if (customAttributes) {
```
I think this check is enough